rise-logger-utils.html

```diff
@@ -0,0 +1,104 @@
+<link rel="import" href="../polymer/polymer.html">
+
+<!--
+`rise-logger-utils` is a web component that is a utilities helper for `rise-logger`.
+
+-->
+
+<script>
+  (function() {
+    /* global Polymer */
+    /* jshint newcap: false */
+
+    "use strict";
+
+    var USAGE_DEV = "dev",
+      USAGE_STANDALONE = "standalone",
+      USAGE_WIDGET = "widget";
+
+    Polymer({
+      is: "rise-logger-utils",
+
+      hostAttributes: {
+        hidden: true
+      },
+
+      _getSuffix: function() {
+        var date = new Date(),
+          year = date.getUTCFullYear(),
+          month = date.getUTCMonth() + 1,
+          day = date.getUTCDate();
+
+        if (month < 10) {
+          month = "0" + month;
+        }
+
+        if (day < 10) {
+          day = "0" + day;
+        }
+
+        return year + month + day;
+      },
+
+      _getUsageType: function(href) {
+        var a = document.createElement("a"),
+          type = USAGE_STANDALONE;
+
+        a.href = href;
+
+        if (a.hostname === "localhost") {
+          type = USAGE_DEV;
+        }
+        else if (a.hostname === "s3.amazonaws.com" && a.pathname.indexOf("widget-") > -1) {
```

Comment:
@stulees We could call it whatever we want. It could just be a boolean named something like `Rise-Widget`. Yeah, true about having to request the file in order to retrieve the headers, but we could cache them or maybe even use local storage since it's not likely to be something that would change often, or ever.

But I'm starting to wonder if we're going a bit too far down a rabbit hole. For example, we'd also have to add something to `circle.yml` to create that metadata automatically on a deploy.

I do think this needs to be addressed, but perhaps for now this is good enough and we create a separate card for working out this piece. What do you think?